core/src/components/select/test/highlight/select.e2e.ts

```diff
@@ -149,6 +149,53 @@ configs({ modes: ['md'], directions: ['ltr'] }).forEach(({ title, screenshot, co
         await expect(container).toHaveScreenshot(screenshot(`select-solid-custom-highlight`));
       });
     });
+  });
+
+  test.describe(title('select: expanded highlight'), () => {
+    test.describe('select: no fill', () => {
+      test('should render bottom highlight', async ({ page }) => {
+        await page.setContent(
+          `
+          <ion-select label="Label" class="select-expanded"></ion-select>
+        `,
+          config
+        );
+
+        const select = page.locator('ion-select');
+        await expect(select).toHaveScreenshot(screenshot(`select-no-fill-highlight`));
+      });
+    });
+    test.describe('select: solid', () => {
+      test('should render bottom highlight', async ({ page }) => {
+        await page.setContent(
+          `
+          <ion-select fill="solid" label="Label" class="select-expanded"></ion-select>
+        `,
+          config
+        );
+
+        const select = page.locator('ion-select');
+        await expect(select).toHaveScreenshot(screenshot(`select-solid-highlight`));
+      });
+    });
+    test.describe('select: outline', () => {
+      test('should render bottom highlight', async ({ page }) => {
+        await page.setContent(
+          `
+          <ion-select fill="outline" label="Label" class="select-expanded"></ion-select>
+        `,
+          config
+        );
+
+        const select = page.locator('ion-select');
+        await expect(select).toHaveScreenshot(screenshot(`select-outline-highlight`));
+      });
+    });
+  });
+});
+
+configs({ modes: ['md', 'ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('select: highlights'), () => {
```

Comment:
As mentioned previously, we should open a new PR that addresses valid and invalid states. This PR should only be for focus.

Reply:
But the md highlight implementation seems to take this under the same feature, using the highlight variable mechanism. Focus can be normal, valid or invalid.